dasharo-security/tpm2-commands.robot

```diff
@@ -225,6 +225,44 @@ TPMCMD011.001 Performing HMAC operation on the file (Ubuntu 22.04)
     Should Contain    ${out1}    hmac.out
     Should Not Contain    ${out2}    hmac.out
 
+TPMCMD012.001 Encrypt and Decrypt non-rootfs partition (Ubuntu 22.04)
+    [Documentation]    Test encrypting and decrypting non-rootfs partition using
+    ...    TPM.
+    # Check:
+    ${out}=    Execute Linux Command    apt info tpm2-tools
+    ${passed}=    Run Keyword And Return Status    "Installed: Yes"
+    ...    Should Contain    ${out}    TPM2_EncryptDecrypt
+    ${out}=    Execute Linux Command    apt info cryptsetup
+    ${passed}=    Run Keyword And Return Status
+    ...    Should Contain    ${out}    "Installed: yes"
+    # Prepare:
+    Execute Linux Command    fallocate -l 20MB test-partition
```

Comment:
@DaniilKl Please read: https://github.com/Dasharo/open-source-firmware-validation#guidelines

Reply:
@macpijan My bad, thanks for pointing out. I have added a commit witch fixes the test so as to match the guidelines: 74e157f93ce40235c75738ceccff1e194c756448. 

I have some CI's failed on my latest commit. Two of the failed CI's report problems with `Secure-Boot` and the other two with `custom boot menu key`. Both of the errors are out of the scope of my changes.